hail_search/hail_search_query.py

```diff
@@ -317,6 +331,46 @@ def _valid_genotype_family_entries(cls, entries, gentoype_entry_indices, genotyp
         )
         return hl.or_missing(is_valid, entries)
 
+    @classmethod
+    def _get_genotype_passes_quality_filter(cls, quality_filter):
+        affected_only = quality_filter.get('affected_only')
+        passes_quality_filters = []
+        for filter_k, value in quality_filter.items():
+            field = cls.GENOTYPE_FIELDS.get(filter_k.replace('min_', ''))
```

Comment:
Is there any reason to format these with `min` if the only applicable filter is `>=`?  We'd never want variants "under" a certain quality right?

Reply:
Unless this is just how the parameters are passed from the frontend, which would make sense.

Reply:
its how they are passed form the frontend, yeah